### PR TITLE
Requested changes

### PR DESCRIPTION
**Pull request by Dmitry-VF**
Requested changes in advise mode

### CODE REVIEW

File: src/twig/Commands/DefaultCommand.cs [hunk 1]
```diff
@@ -52,8 +52,8 @@ public sealed class Settings : CommandSettings
             public bool Verbose { get; set; }
 
             [CommandOption("--advise")]
-            [Description("Find the best compression level given a file and duration")]
-            public int Duration { get; set; }
+            [Description("Find the best compression level given a file and duration\nExample: twig filepath 2000 --advise\nDuration is in milliseconds.")]
```
Comment (mkhomutov) [inline src/twig/Commands/DefaultCommand.cs]: I thought we found a workaround? Or you just did not update the description?

File: src/twig/Commands/DefaultCommand.cs [hunk 2]
```diff
@@ -52,8 +52,8 @@ public sealed class Settings : CommandSettings
...
+            public int Advise { get; set; }
```
Comment (mkhomutov) [inline src/twig/Commands/DefaultCommand.cs]: please, rename it to Duration or AdviceDuration

File: src/twig/Helpers/ArgsHelper.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+{
+    using System;
+    using System.Linq;
+
+    public static class ArgsHelper
+    {
+        public static string[] HandleArgs(string[] args, int defaultVal)
+        {
+            var argsList = args.ToList();
+            if (argsList.Contains("--advise"))
```
Comment (mkhomutov) [inline src/twig/Helpers/ArgsHelper.cs]: too many `if` inside each others, please inverse this one

File: src/twig/Helpers/ArgsHelper.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            {
+                var adviseIdx = argsList.IndexOf("--advise");
+
+                if (argsList.Count > adviseIdx + 1)
+                {
+                    if (Int32.TryParse(argsList[adviseIdx + 1], out var val))
```
Comment (mkhomutov) [inline src/twig/Helpers/ArgsHelper.cs]: think about merging conditions

**Review by mkhomutov — CHANGES_REQUESTED** [review]

File: src/twig/Commands/DefaultCommand.cs (hunk 1 above)
Comment (Dmitry-VF) [reply src/twig/Commands/DefaultCommand.cs]: oops

**Review by mkhomutov — APPROVED** [review]

**Pull request merged**